packages/ynap-parsers/package.json

```diff
@@ -37,6 +37,6 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "build": "npx jest && tsc && cp -r ./lib/* . && rm -rf ./lib",
-    "fetch-bank2ynab": "ynap-bank2ynab-converter -b cf153c196acd809a2e67638df9c8bc157345f567 -o src/bank2ynab/banks.json"
+    "fetch-bank2ynab": "ynap-bank2ynab-converter --branch develop --exclude 'DE Deutsche Kreditbank checking new' --output src/bank2ynab/banks.json"
```

Comment:
I think this should be moved to the ignoreList in https://github.com/envelope-zero/you-need-a-parser/pull/278/files#diff-335e9b3f11ca370983b9cc5bfd9156874bd9d0d66ef34395eae520593e58c754R240?

Reply:
Good catch, the other ignore list was missing the `de` prefix, so it did nothing. Fixed.